xmodule/js/src/capa/display.js

```diff
@@ -696,7 +701,7 @@
         };
 
         Problem.prototype.reset = function() {
-            return this.disableAllButtonsWhileRunning(this.reset_internal, false);
+            return this.disableAllButtonsWhileRunning(this.reset_internal, false, resetClick);
```

Comment:
Thanks for the patience! I finally went through this PR. Can you explain why this method doesn't work correctly? This way, we can better understand why this is the better approach.

And I understand this was opened a while ago, but any info would be very appreciated it! Thank you again